src/lib/autocomplete/autocomplete.ts

```diff
@@ -57,6 +57,11 @@ export class MdAutocomplete implements AfterContentInit {
   /** Whether the autocomplete panel should be visible, depending on option length. */
   showPanel = false;
 
+  _isOpen: boolean = false;
```

Comment:
I'm not sure we need this extra flag. `isOpen` should the same as `this.showPanel && this.options.length > 0`.

Reply:
That was my initial thought as well, however, `showPanel` is not set to false on `blur`, which prevents it from working that way.

Reply:
Makes sense.

Reply:
Cool, also a final nit: can you add a doc string to the `isOpen` getter so it shows up in the docs?

Reply:
Yeah, not sure how I deleted the doc line I had for it.